### ISSUE TITLE
Support sinon@4.0.0

### ISSUE THREAD

**Issue by bjornhanson** [domenic/sinon-chai#112]
After upgrading sinon to v4.0.0, npm complains about sinon-chai missing its peer dependency sinon:

```
$ npm ls > /dev/null
npm ERR! peer dep missing: sinon@^1.4.0 || ^2.1.0 || ^3.0.0, required by sinon-chai@2.13.0
```

**Comment by domenic** [comment]
Yes, as can be seen from #111, the test suite fails with Sinon v4, so this is an accurate warning.

**Comment by domenic** [comment]
Fixed by #111.
